redcap_bridge/utils.py

```diff
@@ -136,24 +136,27 @@ def conversion_csv_to_json(csv_file):
 
     list_of_dict = df.to_dict('records')
     for redcap_field_dict in list_of_dict:
-        if redcap_field_dict['Field Type'] == 'text':
-            if redcap_field_dict['Text Validation Type OR Show Slider Number'] == 'number' or redcap_field_dict['Text Validation Type OR Show Slider Number'] == 'integer':
-                elab_dict = number_to_dict(redcap_field_dict)
-            elif redcap_field_dict['Text Validation Type OR Show Slider Number'] == 'date_dmy':
-                elab_dict = date_to_dict(redcap_field_dict)
+        if redcap_field_dict['Branching Logic (Show field only if...)'] == '':
```

Comment:
Instead of further nesting your conditionals here that might be a prime example to use the `continue` command in python ;)

Reply:
So use a continue right after this if?